src/applications/proxy-rewrite/tests/e2e/proxy-rewrite-entry.cypress.spec.js

```diff
@@ -0,0 +1,55 @@
+import manifest from '../../manifest.json';
+import { proxyRewriteWhitelist as whitelist } from '../../proxy-rewrite-whitelist.json';
```

Comment:
We're not using `whitelist` as a term anymore in the site. Understanding that it's in the filename, but if we can update these instances to `allowlist` please do.